src/MongoDB/WriteConcern.c

```diff
@@ -137,7 +137,7 @@ PHP_METHOD(WriteConcern, getW)
 		PHONGO_RETURN_STRING(PHONGO_WRITE_CONCERN_W_MAJORITY);
 	}
 
-	if (intern->write_concern->w != MONGOC_WRITE_CONCERN_W_DEFAULT) {
+	if (mongoc_write_concern_get_w(intern->write_concern) != MONGOC_WRITE_CONCERN_W_DEFAULT) {
```

Comment:
Journal has an "is_set" method because the public API only returns a boolean; however, w's states were already defined as public API constants, so no method was needed.